views/logon.ejs

```diff
@@ -1,19 +1,21 @@
 <%- include("partials/head.ejs") %>
 <%- include("partials/header.ejs") %>
-<form method="POST">
-  <div>
-    <label name="email">Enter your email:</label>
-    <input name="email" />
+<div class="d-flex flex-column align-items-center justify-content-center" style="width: 100%; height: 70vh;">
+  <div style="width: 30%; padding: 20px; background-color: #fff; box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.1);">
+    <form method="POST" style="margin-top: 20px;">
+      <div class="form-group">
+        <label for="email">Enter your email:</label>
+        <input name="email" id="email" class="form-control" />
+      </div>
+      <div class="form-group">
+        <label for="password">Enter your password:</label>
+        <input type="password" name="password" id="password" class="form-control" />
+      </div>
+      <div>
+        <button class="btn mt-3" style="background-color: #d1e7dd; color: #0c4128; width: 100%;">Login</button>
```

Comment:
Nitpick:
All buttons should have `type=` attributes (in this case a `"submit"`).